build.gradle

```diff
@@ -755,6 +901,20 @@ idea {
                 "2. Run Server"(Gradle) {
                     taskNames = ["runServer"]
                 }
+                "1a. Run Client (Java 17)"(Gradle) {
+                    taskNames = ["runClient17"]
+                }
+                "2a. Run Server (Java 17)"(Gradle) {
+                    taskNames = ["runServer17"]
+                }
+                "1b. Run Client (Java 17, Hotswap)"(Gradle) {
+                    taskNames = ["runClient17"]
```

Comment:
This task doesn't appear in the list as task name conflicts with 1a

Reply:
It appears for me, hmm (when I press the refresh gradle button twice for some reason, it doesn't on the first try)

Reply:
Neither of reloading gradle plugin multiple times nor removing caches works for me

Reply:
It appears for me both in the toolbar and the gradle panel under "Run Configurations" (I'm using the beta new ui so the theme looks a bit different than the default but it's the same ui)
![image](https://user-images.githubusercontent.com/1017004/217523224-510effad-17b8-4dc1-9c8d-8324b2c5d9fc.png)

Reply:
Oh it's there, just not on toolbar.
<img width="282" alt="hs1" src="https://user-images.githubusercontent.com/40035906/217525979-e3f0d749-7704-44fc-bb44-d8a3d16e4dbf.PNG">
<img width="174" alt="hs2" src="https://user-images.githubusercontent.com/40035906/217525987-563f8e0d-1924-47e1-b00e-0eb0f67e8944.PNG">

Reply:
The "modded minecraft" section you're looking at is the gradle task list, not the intellij run configurations list. That one is below:
![image](https://user-images.githubusercontent.com/1017004/217526797-9ad3bae6-34df-488a-bb6e-944e37ceb375.png)

It's possible it doesn't show on your toolbar because you have too many already existing ones, if you click Edit Configurations does it show up there? You can delete the old ones

Reply:
Oh yeah I can see it there, just didn't know it's not at `build` folder